models/localized-string/src/generator.ts

```diff
@@ -0,0 +1,18 @@
+import type { TLocalizedString } from './types';
+
+import { Generator, fake } from '@commercetools-test-data/core';
+
+/**
+ * A localized string is a JSON object where the keys are of ↗ IETF language tag,
+ * and the values the corresponding strings used for that language.
+ */
+const generator = Generator<TLocalizedString>({
+  name: 'LocalizedString',
+  fields: {
+    de: fake((f) => f.lorem.words(5)),
+    en: fake((f) => f.lorem.words(3)),
```

Comment:
Shouldn't `en` be the first?

Reply:
What's the difference?

Reply:
It's a nitpicking, I mean commercetools is an International company and I would assume that English is the first locale. Feel free to ignore.